src/utils/PushNotificationAPI.ts

```diff
@@ -0,0 +1,195 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { Request, Response, Router } from "express"
+import { SensorEventRepository, ParticipantRepository, TypeRepository } from "../repository"
+import { PushNotificationQueue } from "../utils/queue/PushNotificationQueue"
+
+export const PushNotificationAPI = Router()
+
+/**Push notification API--used by external clients to send notifications
+ *
+ *
+ */
+PushNotificationAPI.post("/notifications", async (req: Request, res: Response) => {
```

Comment:
Can you add some documentation around what the input and output are for this API call?